### ISSUE TITLE
Doubled words

### ISSUE THREAD

**Issue by adamchainz** [anthonynosek/sprint-reader-chrome#9]
I'd like to suggest some kind of special treatment for words that appear twice in a row, for example "had" in "He had had a good time". Whenever they appear I have to double-take and normally pause.

My suggestion would be either to try #7 or to treat them as one word with twice the time on screen.

I don't think going beyond doubles (triples) will help much, in fact you'll need to be careful not to die with http://en.wikipedia.org/wiki/James_while_John_had_had_had_had_had_had_had_had_had_had_had_a_better_effect_on_the_teacher or http://en.wikipedia.org/wiki/Buffalo_buffalo_Buffalo_buffalo_buffalo_buffalo_Buffalo_buffalo .

**Comment by anthonynosek** [comment]
Hello Adam,

 

Are you aware Sprint Reader is an open-source project? At the current time I am not working on Sprint Reader due to other commitments. If you wish to view the code or perhaps do some work on Sprint Reader please visit the Sprint Reader page on GitHub.

 

Thanks again for your suggestions.

 

Kind Regards

Anthony Nosek

**Comment by adamchainz** [comment]
Yes I'm aware, I'm just starting a conversation here. I might have a go myself at adding this. Thanks for the reply.

**Comment by anthonynosek** [comment]
Hello Adam,

 

Sure, happy to have a conversation. I just wasn’t aware if you knew Sprint Reader was an open source project. I receive numerous emails regarding Sprint Reader and new functionality which is part of the reason I moved it to open-source.

 

Let me know if you have any questions after reviewing the code.

 

Cheers

Anthony
